### DISCUSSION TITLE
How to gracefully resolve conflicts when parentVersion in the pom file in a multi-module maven project merges code？

### DISCUSSION THREAD

**Discussion by xingyizhu** [aws/aws-sdk-java-v2#4242] [General]
Hi, May I ask that the version numbers of so many pom files will conflict when git merge is executed, do you manually resolve the conflicts one by one?

**Comment by debora-ito** [comment]
This question is not specific to the AWS SDK for Java.

You can search for tutorials about code conflict resolution on developer communities.
